message.go

```diff
@@ -2,14 +2,25 @@ package log
 
 import (
 	"fmt"
+	"os"
 	"time"
+
+	"golang.org/x/term"
 )
 
 const (
-	ResetColor   = "\033[0m"
-	DebugColor   = "\033[0;2m"
-	WarningColor = "\033[1;33m"
-	ErrorColor   = "\033[1;31m"
+	AnsiReset  = "\033[0m"
+	AnsiDim    = "\033[0;2m"
+	AnsiYellow = "\033[1;33m"
+	AnsiRed    = "\033[1;31m"
+)
+
+var (
+	DefaultColor = AnsiReset
+	InfoColor    = AnsiReset
+	DebugColor   = AnsiDim
+	WarningColor = AnsiYellow
+	ErrorColor   = AnsiRed
 )
```

Comment:
That puts me back into the state that @dbutenhof suggested against in a previous comment, where I had `DefaultColor` assigned as a `var` and the rest as `const` because it was the only one I needed to change for the condition of no terminal.